packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -106,6 +98,14 @@ contract Escrow is
     _transferOwnership(msg.sender);
   }
 
+  /**
+   * @notice Returns the storage, major, minor, and patch version of the contract.
+   * @return The storage, major, minor, and patch version of the contract.
+   */
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
```

Comment:
moving this back to above `initialize` for consistency with FA, MTW, GrandaMento